config/packages/oneup_flysystem.yaml

```diff
@@ -7,4 +7,6 @@ oneup_flysystem:
     filesystems:
         datastore_flysystem:
             adapter: data_store_adapter
+            visibility: public
+            directory_visibility: public
```

Comment:
This is a suggestion. I added these to the  default config, because it seemed easier to change, with else code, and easier to maintain in the future.

I have not tested, so please test it, with the other code removed.

Reply:
I searched the codebase for any place else where we were doing a stream or file write and these are the only places. I'm unsure about the syntax here. I came across documentation showing for each of these sections, you could define the octals for what we want to be public vs private for both directories and files, but I didn't try like this. I'll give it a test.

Reply:
Seems to work fine like this.